tests/e2e/utils/index.js

```diff
@@ -1,6 +1,8 @@
+export { activateAMPWithMode, allowedAMPModes, setAMPMode } from './activate-amp-and-set-mode';
```

Comment:
Let's not export `allowedAMPModes` here as it isn't a utility.

It can still be exported from the module in case it's needed; I don't think it is yet though.